sources/HUBViewController.m

```diff
@@ -1258,7 +1258,7 @@ - (void)loadImageFromData:(id<HUBComponentImageData>)imageData
         contextsForURL = [NSMutableArray arrayWithObject:context];
         self.componentImageLoadingContexts[imageURL] = contextsForURL;
         
-        dispatch_async(dispatch_get_main_queue(), ^{
+        HUBPerformOnMainQueue(^{
```

Comment:
Actually, here we want to always run the code async, to make sure that the cell that the image is for is registered with `UICollectionView` before it is loaded. We should add a test for that, that would've failed this change.